sky/data/data_utils.py

```diff
@@ -558,73 +560,112 @@ def get_cos_regions() -> List[str]:
     ]
 
 
-class Rclone():
-    """Static class implementing common utilities of rclone without rclone sdk.
-
-    Storage providers supported by rclone are required to:
-    - list their rclone profile prefix in RcloneClouds
-    - implement configuration in get_rclone_config()
-    """
-
-    RCLONE_CONFIG_PATH = '~/.config/rclone/rclone.conf'
-    _RCLONE_ABS_CONFIG_PATH = os.path.expanduser(RCLONE_CONFIG_PATH)
-
-    # Mapping of storage providers using rclone
-    # to their respective profile prefix
-    class RcloneClouds(Enum):
-        IBM = 'sky-ibm-'
+class Rclone:
+    """Provides methods to manage and generate Rclone configuration profile."""
+
+    class RcloneStores(enum.Enum):
+        S3 = 'S3'
```

Comment:
Can we add R2 as well? Should be very similar to S3 except with a different endpoint.